addons/survey/static/tests/components/question_page_one2many_field_tests.js

```diff
@@ -301,4 +304,59 @@ QUnit.module("QuestionPageOneToManyField", (hooks) => {
             assert.strictEqual(target.querySelector(".o_is_section [name=title]").innerText, "a");
         }
     );
+
+    QUnit.test("full page list question creation", async (assert) => {
+        const indexes = [];
+        const questions = [];
+        for (let i = 1; i <= 45; i++) {
+            questions.push({
+                id: i,
+                is_page: false,
+                title: `question ${i}`,
+                sequence: i,
+            });
+            indexes.push(i);
+        }
+
+        console.log(questions);
+        serverData.models.survey_question.records = questions;
+        serverData.models.survey.records[0].question_and_page_ids = indexes;
+        await makeView({
+            type: "form",
+            resModel: "survey",
+            resId: 1,
+            serverData,
+            arch: `
+            <form>
+                <field name="question_and_page_ids" widget="question_page_one2many">
+                    <tree>
+                        <field name="is_page" invisible="1" />
+                        <field name="title" />
+                        <field name="random_questions_count" />
+                        <field name="sequence" widget="handle"/>
+                    </tree>
+                </field>
+            </form>
+        `,
+            mockRPC(route, args) {
+                if (args.method === "web_save" && args.model === "survey") {
+                    const sequence = args.args[1].question_and_page_ids[0][2].sequence;
+                    assert.step(`question sequence ${sequence}`);
+                }
+            },
+        });
```

Comment:
I may miss something but do we really check that we load the correct page (last page) ?